sfaira/data/dataloaders/base/dataset_group.py

```diff
@@ -673,14 +674,18 @@ def clean_ontology_class_maps(self):
                                     if x not in [
                                         self._adata_ids.unknown_metadata_identifier,
                                         self._adata_ids.not_a_cell_celltype_identifier
-                                    ]
+                                    ] and
```

Comment:
can we refactor this `(not isinstance(x, str) or not x.startswith(self._adata_ids.custom_metadata_prefix))` into a function `def is_custom(x) -> bool` so that it s a bit easier to maintain this repeated check?